bears/yaml/TravisLintBear.py

```diff
@@ -25,6 +28,31 @@ class TravisLintBear:
     CAN_DETECT = {'Formatting', 'Syntax'}
     SEE_MORE = 'https://docs.travis-ci.com/user/travis-lint'
 
+    # IP Address of www.google.com
+    CHECK_CONNECTION_URL = 'http://216.58.218.174'
+    DEFAULT_TIMEOUT = 15
+
+    @classmethod
+    def check_prerequisites(cls):
+        code = cls.get_status_code(
+            cls.CHECK_CONNECTION_URL, cls.DEFAULT_TIMEOUT)
+        if which('travis') is None:  # pragma: no cover
+            return ('Please ensure that travis has been installed. Refer to'
+                    'https://docs.travis-ci.com/user/travis-lint'
+                    '#Command-line-Validation for more details.')
```

Comment:
Can't we use the check of the base class? This message is probably better than the default one, however optimally `requirements` will take this over^^